contracts/coordinator/src/contract/query.rs

```diff
@@ -1,14 +1,42 @@
 use std::collections::HashSet;
 
-use cosmwasm_std::{to_json_binary, Addr, Deps, Order, QueryRequest, StdResult, WasmQuery};
+use cosmwasm_std::{Addr, Deps, Order, StdResult};
+use error_stack::{Result, ResultExt};
 use itertools::Itertools;
-use service_registry_api::msg::QueryMsg::Verifier;
 use service_registry_api::msg::VerifierDetails;
 
 use crate::error::ContractError;
 use crate::msg::VerifierInfo;
 use crate::state::{CONFIG, VERIFIER_PROVER_INDEXED_MAP};
 
+pub fn check_verifier_ready_to_unbond(deps: Deps, verifier_address: Addr) -> StdResult<bool> {
+    Ok(!is_verifier_in_any_verifier_set(deps, &verifier_address))
+}
+
+pub fn verifier_details_with_provers(
+    deps: Deps,
+    service_name: String,
+    verifier_address: Addr,
+) -> Result<VerifierInfo, ContractError> {
+    let config = CONFIG.load(deps.storage).expect("couldn't load config");
```

Comment:
what I meant is create a `state::load_config` function that hides the panic, compare https://github.com/axelarnetwork/axelar-amplifier/blob/58a7ea7f1bf998bee60ec90f66befaebd4fa5c4a/contracts/axelarnet-gateway/src/state.rs#L59